src/app/static/webpack.config.js

```diff
@@ -1,7 +1,32 @@
 const path = require('path');
-const config = require('./webpack.common.config');
```

Comment:
Since we're using jest instead of mocha-webpack now, we didn't need a separate shared `webpack.common.config` file, so I merged it with this one.